src/services/cdn-api-client.js

```diff
@@ -35,6 +35,15 @@ class CdnApiClient {
 
     return request.then(res => res.data);
   }
+
+  deleteFile(prefix, fileName) {
```

Comment:
To reflect the URL, I think this would be better called "deleteCdnObject" or just "deleteObject".

Reply:
Renamed to deleteCdnObject